internal/config/config.go

```diff
@@ -6,5 +6,5 @@
 package config
 
 const (
-	DefaultKyma2Version = "2.20.1"
+	DefaultKyma2Version = "2.20"
```

Comment:
Please change the version to 2.20.0. Kyma follows the semantic versioning and the latest one is 2.20.0
```suggestion
	DefaultKyma2Version = "2.20.0"
```

I also checked it locally and just having `major.minor` does not work.
![Screenshot 2023-12-18 at 08 32 20](https://github.com/kyma-project/cli/assets/48282931/5bb836ef-dbbb-416a-a87e-5b85afe918da)